packages/backend/src/services/UnfurlService/UnfurlService.ts

```diff
@@ -375,8 +376,16 @@ export class UnfurlService {
         const startTime = Date.now();
         let hasError = false;
 
-        const isPuppeteerSetViewportDynamicallyEnabled = false;
-        const isPuppeteerScrollElementIntoViewEnabled = false;
+        const isPuppeteerSetViewportDynamicallyEnabled =
+            await isFeatureFlagEnabled(
+                FeatureFlags.PuppeteerSetViewportDynamically,
+                { userUuid },
```

Comment:
Before we were passing the organization uuid. Is that not needed? 

```
        const isPuppeteerSetViewportDynamicallyEnabled =
            (await postHogClient?.isFeatureEnabled(
                'puppeteer-set-viewport-dynamically',
                userUuid,
                organizationUuid !== undefined
                    ? {
                          groups: {
                              organization: organizationUuid,
                          },
                      }
                    : {},
            )) ?? false;
```

Reply:
Good catch - the helper supports it but it's optional; I should be passing it explicitly here.